build/images/codegen/Dockerfile

```diff
@@ -22,6 +22,8 @@ ARG K8S_VERSION=1.15.0
 ARG KUBEOPENAPI_VERSION=v0.0.0-20190228160746-b3a7cee44a30
 
 RUN go get k8s.io/code-generator/cmd/client-gen@kubernetes-$K8S_VERSION && \
+    go get k8s.io/code-generator/cmd/lister-gen@kubernetes-$K8S_VERSION && \
```

Comment:
@tnqn i plan to add the following to codegen image.. you think it is safe to push the docker image to docker hub.. im not sure who has the permissions.. i can spin these new generators out in a different PR too if you think that would be wise